source/blog/2016-07-20-vagrant-service-manager-install-cli.html.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Download client binaries through vagrant-service-manager in ADB
+author: budhram
+date: 2016-07-20 09:56 UTC
+tags: atomic-developer-bundle, vagrant-service-manager
+published: true
+comments: true
+---
+
+The [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) plugin is designed to enable easier access to the features and services provided by the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). More information can be found in the README  of 'vagrant-service-manager' repo.
+
+This blog is about the new feature in 'vagrant-service-manger' i.e. `install-cli` command. `install-cli` got released with 'vagrant-service-manager' version 1.2.0. This command installs the client binary for any of the services provided by ADB. At this point it can download client binaries for docker and OpenShift.
+
+Usage:
+
+```
+vagrant service-manager install-cli [service]
+
+Where 'service' can be 'docker' and 'openshift'.
+```
+
+Make sure you have 'vagrant-service-manager' plugin installed in your machine. If not, install it via following command as:
+
+```
+$ vagrant plugin install vagrant-service-manager
+```
+
+Now, ensure it has been installed properly with version 1.2.0 by running following command as:
+
+```
+$ vagrant plugin list
+vagrant-service-manager (1.2.0)
+```
+
+Before proceeding let's setup OpenShift Origin using ADB. To setup, run the following commands:
```

Comment:
Here is how you can use the command.  In this example, we will use an ADB set up and running OpenShift Origin.  To setup the ADB and start OpenShift, use the following commands: